Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.py

```diff
@@ -96,6 +159,42 @@ def get_dlp_report(self, report_id: str, fetch_snippets=False):
 
         return self._get_dlp_api_call(url)
 
+    def get_dlp_incidents(self, regions: str, start_time: int = None, end_time: int = None) -> dict:
+        url = INCIDENTS_URL
+        params = {}
+        if regions:
+            params['regions'] = regions
+        if start_time:
+            params['start_timestamp'] = str(start_time)
+        if end_time:
+            params['end_timestamp'] = str(end_time)
+        query_string = urllib.parse.urlencode(params)
+        url = f"{url}?{query_string}"
+        resp, status_code = self._get_dlp_api_call(url)
+        return resp
+
+    def update_dlp_incident(self, incident_id: str, feedback: FeedbackStatus, user_id: str, region: str,
+                            report_id: str, dlp_channel: str):
+        """
+                Update Incident with user provided feedback
+                Args:
+                    incident_id: The id of the incident to update
+                    feedback: 'business_justified', 'true_positive' or 'false_positive'
+                    user_id: The user that initiated the request
+                    region: The DLP region
+                    report_id: The report ID for the incident
+                    dlp_channel: The DLP channel (service name)
+
+                Returns: DLP Incident json
+                """
+        payload = {
+            'user_id': user_id,
+            'report_id': report_id,
+            'service_name': dlp_channel
+        }
+        url = f'{UPDATE_INCIDENT_URL}/{incident_id}?feedback_type={feedback.value}&region={region}'
```

Comment:
Please pass `feedback_type` and `region` via `payload`, it's bad practice to pass it via the URL.

Reply:
This is dictated by the backend API that I have no control of. Can we keep it as it is and make improvements in the next release?

Reply:
@henry-sue-pa The effects should be the same. Does it not work when you pass it via the `payload`?

Reply:
It didn't work when I passed the parameters in the payload.